src/DataProtection/DataProtection/src/Microsoft.AspNetCore.DataProtection.csproj

```
@@ -26,10 +26,14 @@
     <Reference Include="Microsoft.Extensions.Hosting.Abstractions" />
     <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
     <Reference Include="Microsoft.Extensions.Options" />
-    <Reference Include="Microsoft.Win32.Registry" />
     <Reference Include="System.Security.Cryptography.Xml" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' != '$(DefaultNetCoreTargetFramework)' OR
```

This condition means the outer build will get a reference to Microsoft.Win32.Registry. Previous version was wrong too of course.
```suggestion
  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR
      '$(TargetFramework)' == '$(DefaultNetFxTargetFramework)' OR
```